docs/guides/integration/fastapi.md

```diff
@@ -0,0 +1,136 @@
+# Using uv with FastAPI
```

Comment:
I was thinking we'd do "Building a REST API" and "Building a CLI" top-level guides — maybe after "Working on projects" and probably both just using whatever we feel like is good practice e.g. FastAPI and Typer(?). I guess they could go in the integration guides instead, but it seems nice to frame it around the general idea rather than the specific framework (even if we use a specific framework in the example). What do you think?

Reply:
(will review the rest tomorrow)

Reply:
I prefer having something targeted like this. We could always expand it later if needed.